frontend/src/index.js

```diff
@@ -3,12 +3,12 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import * as serviceWorker from './serviceWorker';
 import 'bootstrap/dist/css/bootstrap.css';
-import Navbar from './components/navbar'
+import NavBar from './components/navbar'
 
 
 ReactDOM.render(
   <React.StrictMode>
-      <Navbar />
+      <NavBar />
```

Comment:
Can we change this to <App.js/> and instead put the navbar component inside of the app.js file.

Reply:
I changed it.